### PR TITLE
Mandatory training all job fe

### PR DESCRIPTION
**Pull request by mksrivastava**

### CODE REVIEW

File: src/app/core/model/establishment.model.ts [hunk 1]
```diff
@@ -117,24 +122,27 @@ export interface Establishment {
...
   starters?: Starter[] | string;
   vacancies?: Vacancy[] | string;
+  categories?: Category[] | string;
 }
 
 export enum jobOptionsEnum {
   // tslint:disable-next-line: quotemark
   DONT_KNOW = "Don't know",
   NONE = 'None',
+  ALL = 'all',
```
Comment (aaron-russell) [inline src/app/core/model/establishment.model.ts]: Why are half capitlized and the other half not?

File: src/app/core/model/establishment.model.ts [hunk 2]
```diff
@@ -166,6 +174,27 @@ export interface CancelOwnerShip {
...
+}
+
+export interface mandatoryTraining {
+  trainingCategoryId: number;
+  allJobRoles: boolean;
+  selectedJobRoles?: boolean;
+  jobs: mandatoryJobs[];
+}
+
+export interface mandatoryTrainings {
```
Comment (aaron-russell) [inline src/app/core/model/establishment.model.ts]: I don't think this describes it very well

File: src/app/features/add-mandatory-training/add-mandatory-training.component.ts [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+  private getJobs(): void {
+    this.subscriptions.add(
+      this.jobService
+        .getJobs()
+        .pipe(take(1))
+        .subscribe(jobs => (this.jobs = jobs))
+    );
+  }
+  //get all available trainings
+  private getTrainings(): void {
```
Comment (aaron-russell) [inline src/app/features/add-mandatory-training/add-mandatory-training.component.ts]: Same here, think a different term should be used

File: src/app/core/model/establishment.model.ts (hunk 1 above)
Comment (mksrivastava) [reply src/app/core/model/establishment.model.ts]: Done

File: src/app/core/model/establishment.model.ts (hunk 2 above)
Comment (mksrivastava) [reply src/app/core/model/establishment.model.ts]: Change then ame

File: src/app/features/add-mandatory-training/add-mandatory-training.component.ts (hunk 3 above)
Comment (mksrivastava) [reply src/app/features/add-mandatory-training/add-mandatory-training.component.ts]: Modified.

**Comment by aaron-russell** [comment]
Your build is now failing, other than that, looks okay.

**Pull request merged**